plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -7,35 +7,53 @@
 from hydra.types import ObjectConf
 
 
-class QueueType(Enum):
-    auto = "auto"
+class ExecutorName(Enum):
     local = "local"
     slurm = "slurm"
 
 
 @dataclass
-class SlurmQueueConf:
-    # Params are used to configure sbatch, for more info check:
-    # https://github.com/facebookincubator/submitit/blob/master/submitit/slurm/slurm.py
+class BaseSubmititConf:
+    """Configuration shared by all executors
+    """
 
     # maximum time for the job in minutes
-    time: int = 60
+    timeout_min: Optional[int] = None
     # number of cpus to use for each task
-    cpus_per_task: int = 10
+    cpus_per_task: Optional[int] = None
     # number of gpus to use on each node
-    gpus_per_node: int = 1
+    gpus_per_node: Optional[int] = None
     # number of tasks to spawn on each node
-    ntasks_per_node: int = 1
+    tasks_per_node: Optional[int] = None
+    # memory to reserve for the job on each node (in GB)
+    mem_gb: Optional[int] = None
     # number of nodes to use for the job
-    nodes: int = 1
-    # memory to reserve for the job on each node, in GB
-    mem: str = "${hydra.launcher.mem_limit}GB"
+    nodes: Optional[int] = None
+    # name of the job
+    name: Optional[str] = None
+
+
+@dataclass
+class SubmititSlurmConf(BaseSubmititConf):
```

Comment:
submitit configuration derives from the base configuration so that it can override any default value
and adds a few slurm specific parameters